app/SearchApi/BcGov.Fams3.SearchApi.Core/Adapters/Configuration/ProviderProfileOptions.cs

```diff
@@ -10,5 +10,6 @@ public class ProviderProfileOptions : ProviderProfile
     {
         [Required]
         public string Name { get; set; }
+        public DataProviderEnums DataProviderID { get; set; }
```

Comment:
Why do we have both? Shouldn't it be just the Provider Name (the string value of the enum?)